app/__init__.py

```diff
@@ -194,80 +203,55 @@ def register_blueprint(application):
     )
     from app.user.rest import user_blueprint
 
-    service_blueprint.before_request(requires_admin_auth)
-    application.register_blueprint(service_blueprint, url_prefix="/service")
+    register_notify_blueprint(application, service_blueprint, requires_admin_auth, "/service")
```

Comment:
So if I added all the `_got_registered_once` checks in this file, the formatter complains that the method is too complex. Which is fair. So I refactored out how we create blueprints to the above function.

It is very important that in the lines below we keep the correct auth and prefex for the blueprint. Another set of 👀 going over each of these would be appreciated.